### ISSUE TITLE
Category feature request

### ISSUE THREAD

**Issue by Eliakos** [UserScape/Laravel-Bundles#56]
I suggest to provide the category information into the details of a bundle entry as we currently don't know which category an entry belongs to.

**Comment by ericlbarnes** [comment]
Good idea. I currently have two thoughts. 
1. Make the sidebar category "active" or highlighted.
2. Add some type of breadcrumb. 

I personally think I like #1 better since it wouldn't mess with the structure of the site.

**Comment by Eliakos** [comment]
Yep, i were thinking too of 'Make the sidebar category "active"' or highlighted but will it still be clickable after being activated ?
